.github/CODEOWNERS

```diff
@@ -0,0 +1,160 @@
+*.md @samuelsycamore
+/scripts/ @michaldudak @m4theushw
+
+# Packages
+/packages/mui-base/ @michaldudak @mnajdova
+/packages/mui-docs/ @siriwatknp @flaviendelangle
+/packages/mui-icons-material/ @michaldudak @siriwatknp
+/packages/mui-joy/ @siriwatknp @danilo-leal
+/packages/mui-material/ @mnajdova @danilo-leal
+/packages/mui-system/ @mnajdova @siriwatknp
+
+# Components - Inputs
+/packages/mui-material/src/Autocomplete/ @michaldudak @mnajdova
```

Comment:
Should we include here the components from `mui-base` too?

I would have structured this a bit differently, keeping in mind the long term goal where base would be used in both Joy UI and Material UI, by listing here the components from `/packages/mui-base/*`, and leaving everything in Joy UI and Material UI to the owners of those packages, considering the implementation there is mostly related to the design system. But as this moment this is not the case happy to start with listing both Material UI and MUI Base components.

Reply:
At some point in the future, yes. However, as we're still at a stage where the API shape may change, I prefer to take ownership of Base as a whole so that I can keep an eye on consistency.

Reply:
In this case, it would probably mean that we need both the component owner + you for reviewing @mui/base related changes. I guess this could work 👍 

BTW, I am curious, as more specific paths are assign to most of the packages/mui-material/* paths, when are we going to trigger the packages/mui-material/ owners?

Reply:
> Should we include here the components from mui-base too?

Strong 👍 for this one. It's the best opportunity for people that are exposed to real life challenged surfaced on Material UI to transfer this knowledge to MUI Base as soon as possible. For me the problem I'm concerned about is not the API for developers but end-users.

Reply:
As we discussed at the meeting on Monday, I added just the Base directories.